### PR TITLE
Minor corrections to docs

### PR DESCRIPTION
**Pull request by DePasqualeOrg**

### CODE REVIEW

File: src/docs/content/builders/pin-input.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Pin Input
+title: PIN Input
```
Comment (TGlide) [inline src/docs/content/builders/pin-input.md:2]: Pin Input is fine IMO

File: src/docs/content/builders/dialog.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ Dialogs can be nested. For example, here's a dialog that opens another dialog.
 ## Accessibility
 
 Adheres to the
-[Dialog WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) &
+[Dialog WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) and
```
Comment (TGlide) [inline src/docs/content/builders/dialog.md:146]: We can keep the &s. Makes it quicker to read

File: src/docs/content/builders/pin-input.md (hunk 1 above)
Comment (DePasqualeOrg) [reply src/docs/content/builders/pin-input.md:2]: "Pin" is a word in English with a separate meaning. The correct way to write this is PIN.

File: src/docs/content/builders/dialog.md (hunk 2 above)
Comment (DePasqualeOrg) [reply src/docs/content/builders/dialog.md:146]: It's non-standard and inconsistent.

File: src/docs/content/builders/pin-input.md (hunk 1 above)
Comment (TGlide) [reply src/docs/content/builders/pin-input.md:2]: If we changed it to PIN, we'd also have to rename the method to `createPINInput`, which is quite hard to read. 

Even if Pin is not the most correct form, it's easily understandable in context. We can keep it as it was before.

File: src/docs/content/builders/dialog.md (hunk 2 above)
Comment (TGlide) [reply src/docs/content/builders/dialog.md:146]: I don't mind 🙂

File: src/docs/content/builders/pin-input.md (hunk 1 above)
Comment (DePasqualeOrg) [reply src/docs/content/builders/pin-input.md:2]: You don't have to rename the method. A common convention with acronyms in camel case is to capitalize only the first letter of the acronym.

Comment (TGlide) [reply src/docs/content/builders/pin-input.md:2]: No qualms there! PIN Input it is. Just make sure the sidebar also reads PIN Input 😄 

Also, maybe it'd make sense to rename this to OTP Input in the future. Thoughts @huntabyte ?

Comment (huntabyte) [reply src/docs/content/builders/pin-input.md:2]: I think PIN is more common & recognizable atm!

**Pull request merged**